mopidy/backend.py

```diff
@@ -213,6 +213,20 @@ def translate_uri(self, uri):
         """
         return uri
 
+    def is_live(self, uri):
+        """
+        Defines if the custom URI scheme should be read as a live stream
+
+        *MAY be reimplemented by subclass.*
+
+        It avoids latency on web stream such as radios.
```

Comment:
Personally I'd rather this be clearer and say what it actualy does, which is that it disables buffering. One consequence of that is that it reduces latency, another is that it leaves gaps in your audio.

Reply:
Updated !